integration/images.go

```diff
@@ -414,7 +424,6 @@ func populateVolumeCache() error {
 	warmerCmd := exec.Command("docker",
 		append([]string{
 			"run", "--net=host",
-			"-d",
```

Comment:
previously cache warm-up was happening in the background, that's a nono, we need to have consistent state before moving on.